--- php.ini.orig	2010-10-17 23:12:30.000000000 +0400
+++ php.ini	2010-10-25 01:30:34.000000000 +0400
@@ -437,7 +437,6 @@
 ; Maximum execution time of each script, in seconds
 ; http://php.net/max-execution-time
 ; Note: This directive is hardcoded to 0 for the CLI SAPI
-max_execution_time =300
 
 ; Maximum amount of time each script may spend parsing request data. It's a good
 ; idea to limit this time on productions servers in order to eliminate unexpectedly
@@ -447,7 +446,7 @@
 ; Development Value: 60 (60 seconds)
 ; Production Value: 60 (60 seconds)
 ; http://php.net/max-input-time
-max_input_time = 60
+max_input_time = 6000
 
 ; Maximum input variable nesting level
 ; http://php.net/max-input-nesting-level
@@ -549,7 +548,6 @@
 ; Development Value: On
 ; Production Value: On
 ; http://php.net/log-errors
-log_errors =On
 
 ; Set maximum length of log_errors. In error_log information about the source is
 ; added. The default is 1024 and 0 allows to not apply any maximum length at all.
@@ -806,7 +804,7 @@
 ; http://php.net/extension-dir
 ; extension_dir = "./"
 ; On windows:
-; extension_dir = "ext"
+extension_dir = "c:/php/ext/"
 
 ; Whether or not to enable the dl() function.  The dl() function does NOT work
 ; properly in multithreaded servers, such as IIS or Zeus, and is automatically
@@ -872,7 +870,7 @@
 ; Temporary directory for HTTP uploaded files (will use system default if not
 ; specified).
 ; http://php.net/upload-tmp-dir
-;upload_tmp_dir =
+upload_tmp_dir = "c:\t"
 
 ; Maximum allowed size for uploaded files.
 ; http://php.net/upload-max-filesize
@@ -904,15 +902,17 @@
 
 ; Default timeout for socket based streams (seconds)
 ; http://php.net/default-socket-timeout
-default_socket_timeout = 60
-upload_tmp_dir="C:\WINDOWS\Temp"
+default_socket_timeout = 6000
+;upload_tmp_dir="C:\WINDOWS\Temp"
 session.save_path="C:\WINDOWS\Temp"
+log_errors=On
 error_log="C:\WINDOWS\temp\php-errors.log"
 cgi.force_redirect=0
 cgi.fix_pathinfo=1
 fastcgi.impersonate=1
 fastcgi.logging=0
-extension_dir="c:\PHP\ext"
+max_execution_time=7000
+;extension_dir="C:\PHP\ext"
 
 ; If your scripts have to deal with files from Macintosh systems,
 ; or you are running on a Mac and need to deal with files from
@@ -1562,7 +1562,7 @@
 ; Specified here to create the session id.
 ; http://php.net/session.entropy-file
 ; On systems that don't have /dev/urandom /dev/arandom can be used
-; On windows, setting the entropy_length setting will activate the 
+; On windows, setting the entropy_length setting will activate the
 ; Windows random source (using the CryptoAPI)
 ;session.entropy_file = /dev/urandom
 
@@ -1881,41 +1881,41 @@
 [PHP_MBSTRING]
 extension=php_mbstring.dll
 [PHP_MYSQL]
-extension=php_mysql.dll
+;extension=php_mysql.dll
 [PHP_MYSQLI]
-extension=php_mysqli.dll
+;extension=php_mysqli.dll
 [PHP_OCI8]
-extension=php_oci8.dll
+;extension=php_oci8.dll
 [PHP_OCI8_11G]
-extension=php_oci8_11g.dll
+;extension=php_oci8_11g.dll
 [PHP_OPENSSL]
 extension=php_openssl.dll
 [PHP_PDO_MYSQL]
-extension=php_pdo_mysql.dll
+;extension=php_pdo_mysql.dll
 [PHP_PDO_OCI]
-extension=php_pdo_oci.dll
+;extension=php_pdo_oci.dll
 [PHP_PDO_ODBC]
-extension=php_pdo_odbc.dll
+;extension=php_pdo_odbc.dll
 [PHP_PDO_PGSQL]
-extension=php_pdo_pgsql.dll
+;extension=php_pdo_pgsql.dll
 [PHP_PDO_SQLITE]
-extension=php_pdo_sqlite.dll
+;extension=php_pdo_sqlite.dll
 [PHP_PGSQL]
-extension=php_pgsql.dll
+;extension=php_pgsql.dll
 [PHP_SHMOP]
-extension=php_shmop.dll
+;extension=php_shmop.dll
 [PHP_SOAP]
 extension=php_soap.dll
 [PHP_SOCKETS]
 extension=php_sockets.dll
 [PHP_SQLITE]
-extension=php_sqlite.dll
+;extension=php_sqlite.dll
 [PHP_SQLITE3]
-extension=php_sqlite3.dll
+;extension=php_sqlite3.dll
 [PHP_SYBASE_CT]
-extension=php_sybase_ct.dll
+;extension=php_sybase_ct.dll
 [PHP_TIDY]
-extension=php_tidy.dll
+;extension=php_tidy.dll
 [PHP_XMLRPC]
 extension=php_xmlrpc.dll
 [PHP_XSL]
